src/java.base/share/classes/com/sun/crypto/provider/GHASH.java

```diff
@@ -143,7 +143,7 @@ private static void blockMult(long[] st, long[] subH) {
             throw new ProviderException("Internal error");
         }
         state = new long[2];
-        subkeyHtbl = new long[2*9];
+        subkeyHtbl = new long[2*57]; // 48 keys for the interleaved implementation, 8 for avx-ghash implementation and one for the original key
```

Comment:
nit: the comment is too long, i.e. > 80

Reply:
Ah.. I forgot I didn't change GHASH with my changes.. but I'll fix that thanks